tests/acceptance/test_author_create_typehead.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2016 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this licence, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+from __future__ import absolute_import, division, print_function
+
+import os
+from time import sleep
+
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+
+
+def test_institutions_typehead(selenium, login):
+    """"Test the suggestions in the istitutions field"""
+    selenium.get(os.environ['SERVER_NAME'] + '/submit/author/create')
+    institution_field = selenium.find_element_by_id("institution_history-0-name")
+    force_autocomplete_event(institution_field, "CER", selenium)
+    assert 'CERN' in institution_field.get_attribute("value")
+
+
+def test_experiments_typehead(selenium, login):
+    """"Test the suggestions in the experiments field"""
+    selenium.get(os.environ['SERVER_NAME'] + '/submit/author/create')
+    experiment_field = selenium.find_element_by_id("experiments-0-name")
+    force_autocomplete_event(experiment_field, "ATL", selenium)
+    assert 'ATLAS' in experiment_field.get_attribute("value")
+
+
+def test_advisors_typehead(selenium, login):
+    """"Test the suggestions in the advisors field"""
+    selenium.get(os.environ['SERVER_NAME'] + '/submit/author/create')
+    advisor_field = selenium.find_element_by_id("advisors-0-name")
+    force_autocomplete_event(advisor_field, "alexe", selenium)
+    assert 'Vorobyev, Alexey' in advisor_field.get_attribute("value")
+
+
+def force_autocomplete_event(field, chunk_text, selenium):
+    """Performs selection in a html text field"""
+    field.send_keys(chunk_text)
+    selenium.execute_script('$("#header-section").hide()')
+    selenium.execute_script('$("#collections-section").hide()')
+    WebDriverWait(selenium, 10).until(EC.presence_of_element_located((By.CSS_SELECTOR, "div.tt-suggestion"))).click()
```

Comment:
I have created test cases that are able to provide only one suggestion element. In fact I have tried in debug mode (with `pdb`) to select  all the elements available with the class `tt-suggestion`, and only one was present in the DOM. Anyway I will check again because I am not totally sure about it.